README.md

```diff
@@ -10,8 +10,7 @@ Engineering](https://cs169.saas-class.org).
 [![Test Coverage](https://codeclimate.com/github/csungwon/esaas-engagements/badges/coverage.svg)](https://codeclimate.com/github/csungwon/esaas-engagements/coverage)
 
 [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2070245)
-
-Heroku Link: https://shielded-bastion-61752.herokuapp.com/
+[Heroku Deployment](https://shielded-bastion-61752.herokuapp.com/)
```

Comment:
Maybe before demo day we should change the heroku URL like group 21 did with theirs. Not necessary by any means but looks nice

Reply:
Yeah I thought about that, but the golden repo already took it. (esaas-engagements.herokuapp.com). I can do something similar (e.g. essas-engagemtns-group20) if you guys want.